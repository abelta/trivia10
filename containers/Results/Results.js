import React from 'react';
import { Title, View } from '@shoutem/ui';

const Results = () => (
  <View styleName="flexible">
    <Title>RESULTS</Title>
  </View>
);

export default Results;
