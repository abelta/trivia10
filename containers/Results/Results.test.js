import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter as Router } from 'react-router-native';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
import Results from '.';

const state = {
  game: {
    score: 9,
  },
};

const store = configureStore()(state);

describe('Results', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Router>
        <Results />
      </Router>
    </Provider>,
  ).toJSON();

  it('renders properly', () => {
    expect(tree).toMatchSnapshot();
  });
});
