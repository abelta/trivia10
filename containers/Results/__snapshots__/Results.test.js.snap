// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Results renders properly 1`] = `
<View
  animationOptions={Object {}}
  onLayout={[Function]}
  style={
    Object {
      "flex": 1,
    }
  }
  styleName="flexible"
  virtual={undefined}
>
  <View
    animationOptions={Object {}}
    onLayout={[Function]}
    style={
      Object {
        "flex": 1,
      }
    }
    styleName="flexible"
    virtual={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      animationOptions={Object {}}
      disabled={false}
      ellipsizeMode="tail"
      onLayout={[Function]}
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "#222222",
          "fontFamily": "Rubik-Regular",
          "fontSize": 25,
          "fontStyle": "normal",
          "fontWeight": "normal",
          "lineHeight": 30,
        }
      }
      virtual={undefined}
    >
      You scored
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      animationOptions={Object {}}
      disabled={false}
      ellipsizeMode="tail"
      onLayout={[Function]}
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "#222222",
          "fontFamily": "Rubik-Regular",
          "fontSize": 25,
          "fontStyle": "normal",
          "fontWeight": "normal",
          "lineHeight": 30,
        }
      }
      virtual={undefined}
    >
      9/10
    </Text>
  </View>
  <View
    animationOptions={Object {}}
    onLayout={[Function]}
    style={
      Object {
        "flex": 1,
      }
    }
    styleName="flexible"
    virtual={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      animationOptions={Object {}}
      disabled={false}
      ellipsizeMode="tail"
      onLayout={[Function]}
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "#222222",
          "fontFamily": "Rubik-Regular",
          "fontSize": 20,
          "fontStyle": "normal",
          "fontWeight": "normal",
          "lineHeight": 25,
        }
      }
      virtual={undefined}
    >
      Want to try again?
    </Text>
  </View>
  <View
    animationOptions={Object {}}
    onLayout={[Function]}
    style={
      Object {
        "flex": 1,
      }
    }
    styleName="flexible"
    virtual={undefined}
  >
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      collapsable={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "borderColor": "#ffffff",
          "borderRadius": 2,
          "borderWidth": 1,
          "flexDirection": "row",
          "justifyContent": "center",
          "opacity": 1,
          "paddingLeft": 15,
          "paddingRight": 5,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        animationOptions={Object {}}
        disabled={false}
        ellipsizeMode="tail"
        onLayout={[Function]}
        style={
          Object {
            "backgroundColor": "transparent",
            "color": "#222222",
            "fontFamily": "Rubik-Regular",
            "fontSize": 20,
            "fontStyle": "normal",
            "fontWeight": "normal",
            "lineHeight": 25,
          }
        }
        virtual={undefined}
      >
        PLAY AGAIN
      </Text>
    </View>
  </View>
</View>
`;
